stable/awsebscsiprovisioner/values.yaml

```diff
@@ -26,32 +26,31 @@ registrar:
       repository: "quay.io/k8scsi/csi-node-driver-registrar"
       tag: "v1.1.0"
 
-
 provisioner:
   # True if enable volume scheduling for dynamic volume provisioning
   enableVolumeScheduling: false
   image:
     repository: "quay.io/k8scsi/csi-provisioner"
-    tag: "v1.3.0"
+    tag: "v1.4.0"
 
 attacher:
   image:
     repository: "quay.io/k8scsi/csi-attacher"
-    tag: "v1.2.0"
+    tag: "v1.2.1"
 
 snapshotter:
   # True if enable volume snapshot
   enabled: false
   image:
     repository: "quay.io/k8scsi/csi-snapshotter"
-    tag: "v1.1.0"
+    tag: "v1.2.2"
```

Comment:
as soon as we update  / support to 1.17 we need to switch to `v2.`
https://kubernetes-csi.github.io/docs/external-snapshotter.html